src/script/components/MessagesList/Message/MessageWrapper.tsx

```diff
@@ -101,12 +105,8 @@ export const MessageWrapper: React.FC<MessageParams & {hasMarker: boolean; isMes
     if (firstAsset instanceof Text) {
       const messageId = message.id;
       const messageText = firstAsset.text;
-      const mentions: MentionEntity[] = firstAsset.mentions() && firstAsset.mentions();
+      const mentions: MentionEntity[] = firstAsset.mentions();
```

Comment:
```suggestion
      const mentions = firstAsset.mentions();
```
This will be inferred automatically by TS :)